### DISCUSSION TITLE
Exportar bocetos de figma

### DISCUSSION THREAD

**Discussion by DaviiidSG** [Launch-X-Latam/MisionFrontEnd#137] [Q&A] [answered]
Alguien podría decirme como exportar los bocetos y diagramas que he hecho en figma para subirlos a github o si se pueden importar como imagen? soy nuevo con figma y no encuentro una opción diferente al link para compartirlo

**Comment by federico42o** [comment, marked as answer]
Tene seleccionada la pagina y te salta a la derecha algo asi, fijate que no estes seleccionando ningun frame porque vas a exportar solo ese frame
![image](https://user-images.githubusercontent.com/50332167/193885965-652def0b-e750-4d68-95f6-329146a4cedd.png)

**Reply by DaviiidSG** [reply]
muchas gracias, ya pude hacerlo
